### PR TITLE
graphQl-912: [Customer] CustomerInput improvements

### PR DESCRIPTION
**Pull request by VitaliyBoyko**
### Description (*)
* Marked email with ! in `CustomerInput`.
* Fixed comment in gender r(.
* Fixed comments of taxvat field in `CustomerInput`.

### Fixed Issues (if relevant)
1. #912: CustomerInput improvements

### CODE REVIEW

**Review by lenaorobei — CHANGES_REQUESTED** [review]
Please address `api-functional` tests failures.

File: dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/CreateCustomerTest.php [hunk 1]
```diff
@@ -140,7 +140,8 @@ public function testCreateCustomerIfInputDataIsEmpty()
 
     /**
      * @expectedException \Exception
-     * @expectedExceptionMessage GraphQL response contains errors: Field CustomerInput.email of required type String! was not provided
+     * @expectedExceptionMessage
+     * GraphQL response contains errors: Field CustomerInput.email of required type String! was not provided
```
Comment (lenaorobei) [inline dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/CreateCustomerTest.php]: You can safely remove `GraphQL response contains errors:` part.

Comment (VitaliyBoyko) [reply dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/CreateCustomerTest.php]: Done

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-5960](https://jira.corp.magento.com/browse/ENGCOM-5960) has been created to process this Pull Request
:eight_spoked_asterisk: @lenaorobei, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Pull request merged**